src/test/codewhisperer/util/codeParsingUtil.test.ts

```diff
@@ -63,47 +61,51 @@ describe('RegexValidationForJava', () => {
     })
 })
 
-describe('isTestFileByNameValidation', () => {
-    it('should return true if the file name matches the test filename pattern - Java', () => {
+describe('isTestFileValidation', () => {
```

Comment:
there is no symbol named `isTestFileValidation`. should this be 
```suggestion
describe('isTestFileByName', () => {
```

Reply:
yea renamed it to `isTestFile` sorry didn't cache that